FreeSurfer/FreeSurferPipeline.sh

```diff
@@ -308,7 +318,30 @@ get_options()
 
 	local error_count=0
 
-	# check required parameters
+	# ------------------------------------------------------------------------------
+	#  Compliance check
+	# ------------------------------------------------------------------------------
+	
+	ProcessingMoce=${p_pipelinemode:-HCPStyleData}	
```

Comment:
Typo.

Reply:
Sorry, meant to say use `p_processing_mode` as the internal variable name.

Reply:
Let me just give the intended line to avoid any confusion:
`ProcessingMode=${p_processing_mode:-HCPStyleData}	`

Reply:
fixed.